docs/usage/prometheus-grafana.md

```diff
@@ -0,0 +1,22 @@
+**Prometheus**
```

Comment:
Please follow standard markdown style https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers. Headers are `#` while normal text should not be prefixed by `*`

Reply:
ok, i just pushed change for the Headers, but just to clarify, I am currently using `*` as an unordered list for this doc

Reply:
The top-level header has to be a single `#`, then a level below `##` and subsequently.

Reply:
The entire doc cannot be an un-ordered list. Please see other documentation sites for style reference
- Prysm: https://docs.prylabs.network/docs/contribute/doc-standards
- Lighthouse: https://lighthouse-book.sigmaprime.io/contributing.html

Reply:
I'm not following.  The entire document isn't an un-ordered list.  Everything except for those two headers are.

Reply:
Please re-check the latest file diff.

Reply:
If you prepend a line with `*` it will be displayed as an unordered list. While redacting markdown you can use a wide range or renderers to visualize how your work will look when rendered. You can even build and serve the docs website locally with
```
mkdocs serve
```
to make sure that all pages look as you expect.